typing/gprinttyp.mli

```diff
@@ -20,10 +20,10 @@
 (**
 A type node is printed as
 {[
-    .------------.
-    | <desc>  id |---->
-    |            |--->
-    .------------.
+    .-------------.
+    | <desc>  idˡᵛˡ |---->
+    |             |--->
+    .-------------.
```

Comment:
The ASCII-art alignment is broken with monospace fonts. Maybe just `id^lvl` in ASCII?